tests/unit/r2i/tensorrt/engine.cc

```diff
@@ -0,0 +1,163 @@
+/* Copyright (C) 2018-2020 RidgeRun, LLC (http://www.ridgerun.com)
+ * All Rights Reserved.
+ *
+ * The contents of this software are proprietary and confidential to RidgeRun,
+ * LLC.  No part of this program may be photocopied, reproduced or translated
+ * into another programming language without prior written consent of
+ * RidgeRun, LLC.  The user is free to modify the source code after obtaining
+ * a software license from RidgeRun.  All source code changes must be provided
+ * back to RidgeRun without any encumbrance.
+ */
+
+#include <r2i/r2i.h>
+#include <r2i/tensorrt/engine.h>
```

Comment:
Why are we including these and not the interfaces? Can't we test classes independently?

Reply:
Some of the classes need some initialization in the other modules. These includes allows setting up the data, while only mocking the TensorRT functions